src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/RightCurlyCheck.java

```diff
@@ -221,7 +222,7 @@ else if (shouldCheckLastRcurly) {
         else if (shouldBeOnSameLine(bracePolicy, details)) {
             violation = MSG_KEY_LINE_SAME;
         }
-        else if (shouldBeAloneOnLine(bracePolicy, details)) {
+        else if (shouldBeAlone(bracePolicy, details)) {
```

Comment:
please keep there method with name 'shouldBeAloneOnLine' (only name not a logic), it is the same in naming form as others.